sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -27,28 +32,47 @@ export class AssessmentAESTController extends BaseController {
     private readonly studentAppealService: StudentAppealService,
     private readonly studentAssessmentService: StudentAssessmentService,
     private readonly assessmentControllerService: AssessmentControllerService,
+    private readonly applicationExceptionService: ApplicationExceptionService,
   ) {
     super();
   }
 
   /**
-   * Method to get all requested assessments for a student
+   * Get all requests related to an assessments for a student
    * application, i.e, this will fetch all pending and denied
-   * student appeal for an application.
-   * @param applicationId, application number.
-   * @returns assessment requests for a student application.
+   * student appeals for an application or possible application
+   * exceptions that will prevent the assessment to proceed till
+   * they are approved, for instance, when a document is uploaded
+   * and need to be reviewed.
+   * @param applicationId application number.
+   * @returns assessment requests or exceptions for a student application.
    */
   @Get("application/:applicationId/requests")
   async getRequestedAssessmentSummary(
     @Param("applicationId") applicationId: number,
   ): Promise<RequestAssessmentSummaryAPIOutDTO[]> {
+    const applicationExceptions =
+      await this.applicationExceptionService.getExceptionsByApplicationId(
+        applicationId,
+        ApplicationExceptionStatus.Pending,
+        ApplicationExceptionStatus.Declined,
+      );
+    if (applicationExceptions?.length) {
+      return applicationExceptions.map((applicationException) => ({
```

Comment:
@andrewsignori-aot  does this mean the request will show `applicationExceptions `, if any else `studentAppeal`, if any? not a combination of both?

Reply:
Yes, that is right. They are mutually exclusive. If there is an exception nothing else will be displayed for the history and the workflow will also be stopped at the beginning. Does it make sense?